vendor/modules.txt

```diff
@@ -42,7 +44,7 @@ github.com/imdario/mergo
 github.com/inconshreveable/mousetrap
 # github.com/json-iterator/go v1.1.11
 github.com/json-iterator/go
-# github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.9
+# github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.10
```

Comment:
ok. Before we merge, we need to update the reference to the RTE and scheduler plugin container images.

Reply:
in `pkg/images/consts.go`

Reply:
We're using the latest scheduler-plugin container

Reply:
The last release of scheduler plugins is 20.10, released on August 18. The PR updating the NRT API was merged on Aug, 31 (https://github.com/kubernetes-sigs/scheduler-plugins/commit/eab609df8d6044f6612f7a8ef201752b77bc80cd) so it seems to me that we are neither using the last container nor we have to date a container image for scheduler plugins which includes the 0.0.10 API.

Reply:
We can push an image to https://quay.io/repository/k8stopologyawareschedwg/scheduler-plugins-kube-scheduler
And we're also going to have the same problem with NFD, this is why I asked https://coreos.slack.com/archives/CQNBUEVM2/p1633002760287400

Reply:
This issue need discussion, let's sync offline

Reply:
I can't recall if we followed up and reached a consensus on this?

Reply:
We decided to have a [release flow](https://github.com/k8stopologyawareschedwg/scheduler-plugins/pull/5 ) under the working group only for the scheduler-plugin (not for NFD)